README.md

```diff
@@ -1,33 +1,99 @@
 # reNFT subgraphs
 
-To get started, first, globally install `graph` binary:
+## Getting Started
 
-`yarn global add @graphprotocol/graph-cli`
+First, globally install `graph` binary:
```

Comment:
let's mention the instructions herein work for the following graph binary:

`@graphprotocol/graph-cli/0.49.0 linux-x64 node-v18.16.0`

and mention that it should work for mac and linux ootb